src/filters/frame_skipping_pp.py

```diff
@@ -0,0 +1,52 @@
+from src.filters.abstract_pp import AbstractPP
+from src.utils import image_utils
+from typing import List
+
+
+class frameSkippingPP(AbstractPP):
+
+    def __init__(self, threshold=0.0, compare_foreground=False, 
+                 distance_metric='absolute_difference'):
+        self.threshold = threshold
+        self.compare_foreground = compare_foreground
+        self.distance_metric = distance_metric
+
+    def predict(self, batch) -> List[bool]:
+        prev_frame = None
+        skipFrames = []
```

Comment:
Code style: variable names are snake-case in python.

Reply:
Renamed